CONTRIBUTING.md

```diff
@@ -2,52 +2,59 @@
 
 Pull requests welcome.
 
-## Working on the client locally
+## Setting Up
 
-`npm install --save notifications-node-client`
+### Docker container
 
-## Tests
+This app uses dependencies that are difficult to install locally. In order to make local development easy, we run app commands through a Docker container. Run the following to set this up:
 
-There are unit and integration tests that can be run to test functionality of the client.
+```shell
+make bootstrap-with-docker
+```
 
-To run the unit tests:
+### `environment.sh`
+
+In the root directory of the repo, run:
+
+```
+notify-pass credentials/client-integration-tests > environment.sh
+```
+
+Unless you're part of the GOV.UK Notify team, you won't be able to run this command or the Integration Tests. However, the file still needs to exist - run `touch environment.sh` instead.
+
+## Tests
 
-`make test`
+There are unit and integration tests that can be run to test functionality of the client.
 
-## Integration Tests
+### Unit tests
 
-Before running the integration tests, please ensure that the environment variables are set up.
+To run the unit tests:
 
 ```
-export NOTIFY_API_URL="https://example.notify-api.url"
-export API_KEY="example_API_test_key"
-export FUNCTIONAL_TEST_NUMBER="valid mobile number"
-export FUNCTIONAL_TEST_EMAIL="valid email address"
-export EMAIL_TEMPLATE_ID="valid email_template_id"
-export SMS_TEMPLATE_ID="valid sms_template_id"
-export LETTER_TEMPLATE_ID="valid letter_template_id"
-export EMAIL_REPLY_TO_ID="valid email reply to id"
-export SMS_SENDER_ID="valid sms_sender_id - to test sending to a receiving number, so needs to be a valid number"
-export API_SENDING_KEY="API_team_key for sending a SMS to a receiving number"
-export INBOUND_SMS_QUERY_KEY="API_test_key to get received text messages - leave blank for local development as cannot test locally"
+make test-with-docker
 ```
 
+### Integration Tests
 
 To run the integration tests:
 
-`make integration-test`
+```
+make integration-test-with-docker
+```
+
+## Working on the client locally
 
-The integration tests are used to test the contract of the response to all the api calls, ensuring the latest version of notifications-api do not break the contract of the notifications-node-client.
+```
+npm install --save notifications-node-client
+```
 
 ## Testing JavaScript examples in Markdown
 
 We automatically test that the JavaScript in the documentation examples matched [our linting standards](https://gds-way.cloudapps.digital/manuals/programming-languages/nodejs/#source-formatting-and-linting),
 this also catches some issues that could stop an example from running when copied.
 
-To test the JavaScript for syntax and linting errors run:
-
-`make markdown-standard-test`
-
 You can then fix some issues automatically by running:
```

Comment:
This sentence could maybe do with tweaking now that the example above it has been removed.

Reply:
Oops, yes you're right. I've amended in 3029667dbb0a5f109f360e094b12167e790763fd.